.github/workflows/test.yml

```diff
@@ -7,20 +7,21 @@ jobs:
 
   test:
 
-    name: Test on Ruby ${{ matrix.ruby }}
+    name: Test (Ruby ${{ matrix.ruby }})
 
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         ruby:
-          - 2.1
-          - 2.2
-          - 2.3
-          - 2.4
-          - 2.5
-          - 2.6
-          - 2.7
+          - '2.1'
+          - '2.2'
+          - '2.3'
+          - '2.4'
+          - '2.5'
+          - '2.6'
+          - '2.7'
```

Comment:
These are Ruby versions not floating point numbers. Quote the values so not to confuse the YAML parser.